### DISCUSSION TITLE
Historic hello world?

### DISCUSSION THREAD

**Discussion by jshaffstall** [anvilistas/anvil-labs#57] [Q&A] [answered]
Trying to get a basic historic hello world going, just creating a portable object and trying to historic.create it.  I'm getting an error:

`ErrorLoadingUserCode: Error loading user code: cannot import name 'LOGGER' from 'anvil_extras.server_utils'`

I have both anvil_extras and anvil_labs in the app's dependencies.  What am I missing to let Anvil Labs see Anvil Extras?

**Comment by s-cork** [comment]
Ah this might be a regression. Do you want to make a pr in anvil extras or anvil labs. 

In anvil labs just use the new logging module. Instead of the LOGGER. 

In anvil extras recreate the missing constant LOGGER in the server utils. See the ._timed module in client code for where it now exists. 

No preference from me.

**Comment by s-cork** [comment, marked as answer]
@jshaffstall hopefully that's fixed - i just pushed a change to anvil-labs that should help

**Reply by jshaffstall** [reply]
Yep, that got me past that error.  Now on to working out how historic actually works!
